### ISSUE TITLE
cant claim gas

### ISSUE THREAD

**Issue by MrHvD** [neotracker/neotracker-wallet#7]
Hi, 

It looks like i cant claim gas anymore, it gives an error at point 1:
1. Send NEO to wallet address.
error code: Claiming all GAS failed: Something went wrong. Try refreshing the page or going back to where you were.. Please try again or refresh the page

I've refreshed the page, used different browsers but nothing happened. Last week it got stuck on nr.4: 
4. Wait for confirmation of claim.

So far i have build up gas since the 2nd of January and I cant claim it. Will this issue be fixed soon?

Kind regards,
Hugo

**Comment by cyberjaspreet** [comment]
I have the same issue, it started recently.  It appears the neotracker is stuck after block #11834354 for me... No transactions, no gas claim, no new blocks after that.....

**Comment by MrHvD** [comment]
mine works again, i just recently tried and it got all the gas. Maybe try again in a few days and see how it goes? please leave a comment if it works, maybe its just a temporary issue that fixes itself

**Comment by cyberjaspreet** [comment]
A couple of days is bad... I will update a comment, whenever it works..... Did you try anything special other than retrying?

**Comment by cyberjaspreet** [comment]
Inam not sure what happened, but I got my Neo where I wanted them to go.  However, the neotracker is still stuck with my old balance.  Will update when I get my GAS

**Comment by cyberjaspreet** [comment]
I got all, for all the users with this problem.... It will work automatically in a couple couple of days......

**Comment by scottreadman** [comment]
I'm having exactly the same issue now. Just signed up today and it's stopped allowing me to claim after an hour. I'm getting came error: 

"Claiming all GAS failed: Something went wrong. Try refreshing the page or going back to where you
were.. Please try again or refresh the page."

Can someone tell me if they found a fix or heard anything from neotracker? I've tweeted them and heard nothing. Should I move my neo elsewhere?

**Comment by illtellyoulater** [comment]
This doesn't look good...

**Comment by MrHvD** [comment]
it works - just sometimes it doesnt work, often it does. check again tomorrow, i claimed gas yesterday and all was fine

**Comment by illtellyoulater** [comment]
Yes but, if Neotracker is an offline wallet that only interacts with the NEO blockchain, then why one day it works and the next it doesn't?

**Comment by Vishal19955** [comment]
The best projects...
